src/project_types/sequence_section.rs

```diff
@@ -12,37 +12,23 @@ use project_types::User;
 #[derive(Clone, Debug, PartialEq, Eq, RustcEncodable, RustcDecodable)]
 pub struct SequenceSection {
     pub seq_name: String,
-    pub index: u32,
+    pub index: u32,  // Starts at 1
+    pub path: String,
     pub num_frames: u32,
     pub editor: Option<User>,
```

Comment:
I don't think the SeqSec should know who its editor is. Its not a source of truth, because the actual permissions take precendence. And I don't think there is any point in caching it here. Your thoughts?